### PR TITLE
New Belarusian Cheat Sheet

### PR DESCRIPTION
**Pull request by qqq3**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [x] New Instant Answer
    - [x] Cheat Sheets: **`New Belarusian Cheat Sheet`**

###### Description of changes
Add Belarusian Cheat Sheet

---

Instant Answer Page: https://duck.co/ia/view/belarusian_cheat_sheet/

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @qqq3

### CODE REVIEW

**Comment by daxtheduck** [comment]
@qqq3 Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please head to https://quackslack.herokuapp.com/ to get an invite.

>More Info: http://docs.duckduckhack.com/submitting/submitting-overview.html

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by edgesince84** [comment]
Thanks, Eugene!

**Comment by mintsoft** [comment]
@zekiel @bill I think a TM needs to grab someone for this?

**Comment by zekiel** [comment]
@mintsoft  on it!

File: share/goodie/cheat_sheets/json/language/belarusian.json [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+                "key": "Tuesday",
+                "val": "Аўторка"
+            },
+            {
+                "key": "Wednesday",
+                "val": "Серада"
+            },
+            {
+                "key": "Thursday",
+                "val": "Чацьвер"
```
Comment (denisbalyko) [inline share/goodie/cheat_sheets/json/language/belarusian.json]: maybe? Чацвер

File: share/goodie/cheat_sheets/json/language/belarusian.json [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+            }
+        ],
+        "Days of the Week": [
+            {
+                "key": "Monday",
+                "val": "Панядзелак"
+            },
+            {
+                "key": "Tuesday",
+                "val": "Аўторка"
```
Comment (denisbalyko) [inline share/goodie/cheat_sheets/json/language/belarusian.json]: maybe? Аўторак

File: share/goodie/cheat_sheets/json/language/belarusian.json [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+
+    "sections": {
+        "Basics": [
+            {
+                "key": "Hello",
+                "val": "Вітаю"
+            },
+            {
+                "key": "Welcome!",
+                "val": "Сярдэчна запрашаем!"
```
Comment (mialeska) [inline share/goodie/cheat_sheets/json/language/belarusian.json]: should be "Сардэчна запрашаем!"

File: share/goodie/cheat_sheets/json/language/belarusian.json [hunk 4]
```diff
@@ -0,0 +1,248 @@
...
+            },
+            {
+                "key": "How are you?",
+                "val": "Як справы?"
```
Comment (mialeska) [inline share/goodie/cheat_sheets/json/language/belarusian.json]: should be "Як маешся?"

File: share/goodie/cheat_sheets/json/language/belarusian.json [hunk 5]
```diff
@@ -0,0 +1,248 @@
...
+                "key": "Fine, thank you",
+                "val": "Добра, дзякуй"
+            },
+            {
+                "key": "What is your name?",
+                "val": "Як вас клічуць?"
+            },
+            {
+                "key": "My name is ...",
+                "val": "Мяне клічуць ..."
```
Comment (mialeska) [inline share/goodie/cheat_sheets/json/language/belarusian.json]: better to say "Мяне зваць ..."

File: share/goodie/cheat_sheets/json/language/belarusian.json [hunk 6]
```diff
@@ -0,0 +1,248 @@
...
+                "key": "Call the Police!",
+                "val": "Патэлефануйце ў паліцыю!"
+            },
+            {
+                "key": "Call an ambulance!",
+                "val": "Выклічце хуткую!"
+            },
+            {
+                "key": "I need a doctor!",
+                "val": "Мне патрэбны доктар!"
```
Comment (mialeska) [inline share/goodie/cheat_sheets/json/language/belarusian.json]: should be "Мне патрэбен доктар!"

**Comment by zekiel** [comment]
We put a call out on Twitter today for this: https://twitter.com/duckduckgo/status/739797998809559040 

Thank you for the help @mialeska!

File: share/goodie/cheat_sheets/json/language/belarusian.json [hunk 7]
```diff
@@ -0,0 +1,248 @@
...
+            }
+        ],
+        "Eating": [
+            {
+                "key": "A glass of water please",
+                "val": "Шклянку вады, калі ласка"
+            },
+            {
+                "key": "A table for one/two please!",
+                "val": "Табліца для аднаго/дзьвух чалавек, калі ласка!"
```
Comment (mialeska) [inline share/goodie/cheat_sheets/json/language/belarusian.json]: should be "Стол для аднаго/двух чалавек, калі ласка!"

File: share/goodie/cheat_sheets/json/language/belarusian.json [hunk 8]
```diff
@@ -0,0 +1,248 @@
...
+                "key": "I am allergic!",
+                "val": "У мяне алергія!"
+            },
+            {
+                "key": "I'm a vegetarian",
+                "val": "Я вегетарыянец"
+            },
+            {
+                "key": "Do you have vegetarian dishes?",
+                "val": "У вас ёсць вегетарыянскій стравы?"
```
Comment (mialeska) [inline share/goodie/cheat_sheets/json/language/belarusian.json]: should be "Ці ёсць у вас вегетарыянскія стравы?"

File: share/goodie/cheat_sheets/json/language/belarusian.json [hunk 9]
```diff
@@ -0,0 +1,248 @@
...
+            },
+            {
+                "key": "Do you have Kosher dishes?",
+                "val": "У вас ёсць Кашэрныя стравы?"
+            },
+            {
+                "key": "Main dish",
+                "val": "Асноўнае страва"
```
Comment (mialeska) [inline share/goodie/cheat_sheets/json/language/belarusian.json]: should be "Асноўная страва"

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Belarusian Cheat Sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/belarusian_cheat_sheet_mobile.png?NkpMOxSOg8)

File: share/goodie/cheat_sheets/json/language/belarusian.json (hunk 2 above)
Comment (qqq3) [inline share/goodie/cheat_sheets/json/language/belarusian.json]: Да, вы правы, спасибо :)

File: share/goodie/cheat_sheets/json/language/belarusian.json (hunk 4 above)
Comment (qqq3) [inline share/goodie/cheat_sheets/json/language/belarusian.json]: @mialeska "How are you?" - "Как дела?" (ну или как вариант "Как ты?"). Разве мой вариант не коректен?

Comment (denisbalyko) [inline share/goodie/cheat_sheets/json/language/belarusian.json]: "Як ты?"

**Comment by edgesince84** [comment]
@mialeska thanks for all your help!

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Belarusian Cheat Sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/belarusian_cheat_sheet_mobile.png?Dmzt2783HS)

**Comment by tagawa** [comment]
@denisbalyko Thank you for the extra comments.

Thanks for implementing the feedback @qqq3. This is now live on the beta server: https://beta.duckduckgo.com/?q=Belarusian+Cheat+Sheet&ia=cheatsheet&iax=1

If we get more feedback on Twitter I'll let you know, otherwise we should be able to merge this soon.

**Comment by tagawa** [comment]
We haven't had any further feedback so I think this is good to go. Thanks again @qqq3!
I see you have other Instant Answers as well. If you're looking for more inspiration, feel free to check out the programming IA coverage table for ideas: https://github.com/duckduckgo/duckduckgo/wiki/Programming-IA-Coverage

**Comment by daxtheduck** [comment]
@qqq3 this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Pull request merged**